bpmnconstraints/mermaid/mermaidtranslation.py

```diff
@@ -0,0 +1,77 @@
+from bpmnconstraints.utils.constants import *
+
+DEFAULT_DIRECTION = "LR"
+SEQUENCE_FLOW = "-->"
+END_EVENT_STYLING_DEF = "classDef EndEvent fill:stroke:#000,stroke-width:4px"
+END_EVENT_STYLE = ":::EndEvent"
+
+
+class Mermaid:
```

Comment:
Consider adding some docstrings here. I suppose in-code documentation wasn't a prio in general so far...